python/cudf/cudf/groupby/groupby.py

```diff
@@ -26,6 +27,20 @@ def dataframe_from_columns(cols, index_cols=None, index=None, columns=None):
 
 
 class _Groupby(object):
+    def serialize(self):
+        header = {}
+        header["groupby"] = pickle.dumps(self)
```

Comment:
@kkraus14 I'm somewhat confused about why this works.  As best as I can tell this is what the previous code did.  That is, simply use pickle to serialize the groupby object.  I was expecting needing to pull out the underlying column buffers stored in `_df`

Reply:
@kkraus14 and I discussed this -- we should passing along the underlying dataframe/series in the serialization process